playroom/components/index.tsx

```diff
@@ -174,9 +174,13 @@ const PreviewToolsControls = React.forwardRef<HTMLDivElement, PreviewToolsContro
                             }}
                             render={({checked}) => (checked ? <IconSun size={24} /> : <IconMoon size={24} />)}
                         />
-                        <IconButton aria-label="Edit in Playroom" size={24} onPress={onEditStoryPress}>
+                        <Touchable
```

Comment:
I think we can change the design here and use new IconButtons

Reply:
Agree. I've updated it :)